lib/string_reverse.rb

```diff
@@ -20,3 +20,12 @@ def string_reverse(my_string)
 
   my_string << "not implemented"

```

Comment:
Because the function returns on line 19, line 21 is what is called "dead code". Line 21 will never get executed. 
Also, you don't want line 21 to be executed either. :-)
So, you can safely delete line 21.